pkg/controllers/cloud/cloudentityselector_controller.go

```diff
@@ -133,132 +137,39 @@ func (r *CloudEntitySelectorReconciler) processCreateOrUpdate(selector *cloudv1a
 
 	cloudInterface, err := cloudprovider.GetCloudInterface(common.ProviderType(accPoller.cloudType))
 	if err != nil {
-		if !preExists {
-			_ = r.processDelete(selectorNamespacedName)
-		}
+		_ = r.processDelete(selectorNamespacedName)
 		return err
 	}
 
 	err = cloudInterface.AddAccountResourceSelector(accPoller.namespacedName, selector)
 	if err != nil {
-		if !preExists {
-			_ = r.processDelete(selectorNamespacedName)
-		}
-		r.Log.Info("selector add failed", "selector", selectorNamespacedName,
-			"poller-exists", preExists)
+		_ = r.processDelete(selectorNamespacedName)
+		r.Log.Info("selector add failed", "selector", selectorNamespacedName)
 		return err
 	}
 
-	if !preExists {
```

Comment:
as you have removed from here, that means poller will always be created with cpa add?

Reply:
yes, thats right. We are checking for poller presence in CES before these calls and making sure it exists, else we do not proceed.